src/components/general/nomenclatural/Class.tsx

```diff
@@ -0,0 +1,23 @@
+/* Import Types */
+import { ReactElement } from "react";
+
+
+/* Props Typing */
+interface Props {
+    classProp: string
+};
+
+
+const Class = (props: Props) => {
+    const { classProp } = props;
+
+    /* Base variables */
+    let classElement: ReactElement;
+
+    /* Format Class according to Nomenclatural Rules */
+    classElement = <span> {classProp ?? 'undefined'} </span>
```

Comment:
Don't think this should be `undefined` but be left empty. If it stays undefined we should keep it in line with the other Undefined which I believe have a capital

Reply:
All 'Undefined' string are now empty strings, also this one with the lowercase 'u'